src/sourcerer/domain/storage_provider/services.py

```
@@ -89,13 +89,14 @@ def delete_storage_item(self, storage: str, key: str) -> None:
 
     @abstractmethod
     def upload_storage_item(
-        self, storage: str, source_path: Path, dest_path: str | None = None
+        self, storage: str, storage_path:str, source_path: Path, dest_path: str | None = None
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)